crates/script/src/lib.rs

```diff
@@ -217,6 +217,10 @@ pub struct ScriptArgs {
 
     #[command(flatten)]
     pub retry: RetryArgs,
+
+    /// Gas per pubdata
+    #[clap(long = "zk-gas-per-pubdata", value_name = "GAS_PER_PUBDATA")]
+    pub gas_per_pubdata: Option<u64>,
```

Comment:
```suggestion
    pub zk_gas_per_pubdata: Option<u64>,
```
Since this is an upstream struct maybe we should prefix with `zk`